types/index.d.ts

```diff
@@ -10,40 +10,21 @@ declare namespace VueI18n {
   interface LocaleMessageArray { [index: number]: LocaleMessage; }
   interface LocaleMessages { [key: string]: LocaleMessageObject; }
   type TranslateResult = string | LocaleMessages;
-  interface DateTimeFormatOptions {
-    year?: string;
-    month?: string;
-    day?: string;
-    hour?: string;
-    minute?: string;
-    second?: string;
-    weekday?: string;
-    hour12?: boolean;
-    era?: string;
-    timeZone?: string;
-    timeZoneName?: string;
-    localeMatcher?: string;
-    formatMatcher?: string;
-  }
-  interface DateTimeFormat { [key: string]: DateTimeFormatOptions; }
-  interface DateTimeFormats { [key: string]: DateTimeFormat; }
+
+  interface DateTimeFormat { [type: string]: Intl.DateTimeFormatOptions; }
```

Comment:
Where `Intl` type definition ?

Reply:
It's included in standard TS types and represents the internationalization options for `toLocaleString` functions.
https://github.com/Microsoft/TypeScript/blob/eb2297df022f6b1f284aff96e93c06097bb01ea5/lib/lib.es5.d.ts#L4093

Reply:
Ok, I can see where you're coming from on this one, I'll revert it rn.